core/database/arango/arango_unix_client.py

```
@@ -0,0 +1,157 @@
+    """
+
+    def __init__(self):
+        """Initialize the Unix socket HTTP client."""
+        try:
+            import requests_unixsocket
+            self.requests_unixsocket = requests_unixsocket
+        except ImportError:
+            raise ImportError(
+                "requests-unixsocket not installed. "
+                "Run: pip install requests-unixsocket"
+            )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)